### DISCUSSION TITLE
Optimum settings for training model on a aesthetic style

### DISCUSSION THREAD

**Discussion by arthurjwalsh94** [TheLastBen/fast-stable-diffusion#1566] [Q&A]
Hi,

What are the optimum settings for training the model on style (mid-journey of some abstract landscapes)? I can't quite figure it out from the notes in the notebook, I have looked at other people's discussions but the answers seem to change as I guess every situation is slightly different.

I am using the Deforum notebook to create an animation of the landscape. So far during the animation, it loses its shape quite quickly, but the initial image can be very good and in keeping with the style I want. I'd like it to be more consistent throughout the animation.

I am using:

30 instance images 
200 Concept images
1000 Unet_trainingsteps
5e-5 Unet_learningrate
Everything is set to default pretty much

Should I select the style_training tick box?

Here is my notebook: 
https://colab.research.google.com/drive/1dep55Pi_KuD0oVYDYHKshqtWAxoGqLoN#scrollTo=LC4ukG60fgMy

Any help appreciated. 

Thanks
Arthur

**Comment by TheLastBen** [comment]
Hi,
If it's a painting style, all you need is 5-10 instance image, no concept images, total text encoder steps = 300 at 1e-6, Unet steps 600-2000 at 5e-6, no need to check other boxes.

**Reply by arthurjwalsh94** [reply]
Legend its looking good. Thank you

**Reply by arthurjwalsh94** [reply]
@TheLastBen - I created this in deforum using the values you mentioned above. 

https://twitter.com/Art_Love_xx/status/1623715463342555136?s=20&t=a6ODnsVAeUiRPS0W3qNkQg

It's pretty good, if you look at the very first frame that's the type of image it's trained on. It goes more illustrative in the animation, which is cool, but do you know if it's possible to tweak it to give it that more digital and clean look like in the first frame? 

If you have a moment, would you mind checking it out to see if you notice anything that I can improve in the model training please?

Also, is there a discord where we can discuss parameters and get feedback etc?

Thanks for all your help
